experimental/auth-logic/wrappers/rekor_wrapper_test.go

```diff
@@ -15,29 +15,86 @@
 package wrappers
 
 import (
+	"io/ioutil"
 	"testing"
+
+	"github.com/go-openapi/strfmt"
 )
 
 const testRekorLogPath = "experimental/auth-logic/test_data/rekor_entry.json"
+const testPubKeyPath = "experimental/auth-logic/test_data/product_team_key.pub"
+const testUnexpiredEndorsementFilePath = "experimental/auth-logic/test_data/oak_endorsement.json"
 
 func TestRekoLogWrapper(t *testing.T) {
 	logEntryAnon, err := getLogEntryAnonFromFile(testRekorLogPath)
```

Comment:
All of these functions are already called in `EmitStatement` via `EmitStatementAs`, so they seem redundant to me. If the idea is to have unit-tests for each function, then each should be a separate function, e.g., `func TestGetLogEntryAnonFromFile((t *testing.T) {`.